build.gradle

```diff
@@ -60,6 +60,7 @@ allprojects {
 
     repositories {
         mavenCentral() { metadataSources { mavenPom(); ignoreGradleMetadataRedirection() } }
+        mavenLocal()
```

Comment:
reminder to remove this once https://github.com/palantir/conjure-java-runtime-api/pull/1352 merges.